index.js

```diff
@@ -30,11 +33,90 @@ module.exports = function(filename, root, visited) {
 
   var results = traverse(filename, root, visited);
   results = removeDups(results);
+  var tree = {
```

Comment:
For the object form, I've always envisioned a structure like:
```js
{
  'a.js': {
    'b.js': {
      'e.js': {}
    },
    'c.js': {}
  }
}
```

Instead of having artificial keys like `root` and `nodes`. It's also better for lookups when you have the filepath as the index since it's likely that I'll want to find subtrees as quickly as possible.